src/components/popover/popover.test.js

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { Popover } from './popover';
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useState: jest.fn(),
+}));
+
+const setState = jest.fn();
+
+describe('Popover', () => {
+  beforeEach(() => {
+    useState.mockImplementation(init => [init, setState]);
+  });
+
+  it('Renders button', () => {
+    render(
+      <Popover />
+    );
+
+    const button = screen.getByRole('button');
+
+    expect(button).toBeInTheDocument();
+  });
+
+  it('Renders div element with close button', async () => {
```

Comment:
It doesn't matter div it or span. Let's change it to drop element because it's the element that will be opened by click.

Reply:
Changed it to drop element.